ohara-manager/client/src/hooks/restartWorkspaceHooks.js

```diff
@@ -73,3 +74,14 @@ export const useRestartWorkspaceAction = () => {
     [dispatch],
   );
 };
+
+export const useResetClustersAction = () => {
```

Comment:
```suggestion
export const useRefreshWorkspaceAction = () => {
```

and move into workspaceHooks.js

Reply:
ok